jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/annotations/Linter.kt

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.cloudformation.annotations
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.type.TypeFactory
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.util.ExecUtil
+import software.aws.toolkits.core.utils.error
+import software.aws.toolkits.core.utils.getLogger
+import software.aws.toolkits.core.utils.info
+import software.aws.toolkits.jetbrains.core.executables.CfnLintExecutable
+import software.aws.toolkits.resources.message
+
+class Linter {
+
+    companion object {
+        private val LOG = getLogger<Linter>()
+        private val mapper = ObjectMapper()
```

Comment:
might make more sense to use the `com.fasterxml.jackson.module.kotlin.jacksonObjectMapper` which is more Kotlin aware - then you can model your objects as data classes

It's already on the classpath, example : https://github.com/aws/aws-toolkit-jetbrains/blob/63b04161077b7fd9f9405075b93f45607ee95445/jetbrains-core/src/software/aws/toolkits/jetbrains/services/RoleValidation.kt#L11